crates/common/chain/lean/src/lean_chain.rs

```diff
@@ -231,3 +239,23 @@ impl LeanChain {
             .cloned()
     }
 }
+
+impl Default for LeanChain {
+    fn default() -> Self {
+        let ream_dir = setup_data_dir("lean_node", None, true)
+            .expect("Unable to initialize database directory");
+        let ream_db =
+            ReamDB::init_lean_db(ream_dir.clone()).expect("unable to init Ream Lean Database");
+        Self {
+            store: Arc::new(Mutex::new(ream_db)),
+            chain: HashMap::new(),
+            post_states: HashMap::new(),
+            known_votes: Vec::new(),
+            head: B256::default(),
+            safe_target: B256::default(),
+            new_votes: Vec::new(),
+            genesis_hash: B256::default(),
+            num_validators: 0,
+        }
+    }
+}
```

Comment:
hmm I think we should just remove Default for LeanChain now, as it doesn't make much sense to be initializing the database like this, it is kind of a trap

Reply:
What do you mean by enabling this? We should remove default and just create LeanChain manually in the test